### PR TITLE
Test coverage

### PR DESCRIPTION
**Pull request by zappen999**
Refactored to be able to test core features. Sorry for big PR 🌮

### CODE REVIEW

File: src/bootstrap/cache.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  enabled: Boolean(process.env.CACHE_ENABLED)
+});
+
+// For testing purposes
+module.exports = {
+  _readthroughWrite: readthroughWrite,
+  _readthroughRead: readthroughRead,
+  _handleRedisError: handleRedisError,
+  _notifyCacheDisabled: notifyCacheDisabled,
+  _redisClient: redisClient
```
Comment (JoelRoxell) [inline src/bootstrap/cache.js:86]: Why are you underscoring these properties?

File: src/schema/resolvers/post.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+/**
+ * Gets list of posts
+ * @return {Promise} List of posts
+ */
+function getPosts() {
+  return Promise.resolve([]);
+}
+
+module.exports = {
+  getPosts: getPosts
```
Comment (JoelRoxell) [inline src/schema/resolvers/post.js:12]: Should be able to use ES6 shorthands

``` js
module.exports = {
  getPosts
}
```

**Review by JoelRoxell — APPROVED** [review]

File: src/bootstrap/cache.js (hunk 1 above)
Comment (zappen999) [reply src/bootstrap/cache.js:86]: It is a common practice to prefix with underscore if the properties are considered "private". In this case, they are for testing purposes only.

**Review by LinusLjung — COMMENTED** [review]
http://giphy.com/gifs/aint-nobody-got-time-for-that-gif-hfvihQ6LF9x2o

**Pull request merged**